### PR TITLE
Initial commit with template :tada:

### PR DESCRIPTION
**Pull request by urielsalis**

### CODE REVIEW

File: public/logo_small.jpg
Comment (anyulled) [inline public/logo_small.jpg:1]: The Logo should be on PNG version, check https://drive.google.com/file/d/18J0fMmZ50RWfPBp5hQbKZIes-5bNep9t/view?usp=drive_link

File: src/content/showcase/_images/astro-docs.png
Comment (anyulled) [inline src/content/showcase/_images/astro-docs.png:1]: WE should remove these images

**Review by anyulled — APPROVED** [review]

**Comment by anyulled** [comment]
- [ ] We should integrate with Netlify for temporal deployment of each pull request

**Comment by urielsalis** [comment]
its even easier, we can just enable the github page

**Pull request merged**